redesign.php

```diff
@@ -25,7 +25,26 @@
             </div>
             <div class="hero-text">
                 <h4>Games let you try on a new identity.</h4>
-                <p>Games are an important tool for teachers because they allow players to take on the role of scientists, journalists, explorers, and more.</p>
+                <p>Games are an important tool for teachers because they allow players to take on the role of
+                    scientists, journalists, explorers, and more.</p>
+            </div>
+        </div>
+    </section>
```

Comment:
Should add the `<div class="bottom-frame"></div>` separator between these levels.